src/main/java/org/cryptomator/ui/unlock/UnlockWorkflow.java

```diff
@@ -107,6 +119,7 @@ protected void succeeded() {
 			case ASK -> Platform.runLater(() -> {
 				window.setScene(successScene.get());
 				window.show();
+				centerOnPrimaryStage(window);
```

Comment:
can we center first and then show the window? (aka setting the position before unhiding the window)

Reply:
Unfortunately, no, the issue arises only after .show() is called. Therefore, the dialog's position must be corrected after it is displayed.